tests/scripts/all.sh

```diff
@@ -2685,7 +2685,17 @@ config_psa_crypto_config_accel_ecc_no_bignum() {
     scripts/config.py unset MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED
     scripts/config.py unset MBEDTLS_KEY_EXCHANGE_ECDH_RSA_ENABLED
 
-    # TODO: DHM and its reverse deps
+    # Disable FFDH because it also depends on BIGNUM.
```

Comment:
Following the point you made on `driver-only-builds.md`, I'm now wondering if we should really disable FFDH or just accelerate it... See [discussion on Slack](https://arm-ce.slack.com/archives/C01CC4Y1T1P/p1690448228276219). For this PR, let's stick with disabling, and we'll see later.

Reply:
Ok understood. It seems that for now nobody replied to that thread yet, but in case anyone does we can decide later if it's worth to include tests for FFDH as well in this PR or create a specific PR for it in this EPIC.